paper/writeup.tex

```diff
@@ -109,14 +109,21 @@
     it is an old observation that the ``junction'' between two distinct haplotypes
     produced by recombination is inherited as a Mendelian marker.
     In this paper, we describe how this recombination-mediated information
-    can in many cases be recovered from inference based solely on polymorphic markers,
-    which produces unary nodes along the branches of the marginal genealogical trees.
+    can in many cases be recovered from inference based solely on polymorphic markers.
+    % NSP-22NOV: "unary node" and "ARG" are not defined so I tried to provide better context here
+    In a genealogical context, this information reflects the persistence
+    of ancestral haplotypes across marginal genealogical trees where they
+    are not coalescence events.
+    We show how these non-coalescing haplotypes (``unary nodes'') may be
+    inserted into the ancestral recombination graph (ARG), a compact
+    but information-rich data structure describing the genealogical
+    relationships among recombinant sequences.
     The resulting inferred ARGs are smaller, faster to compute with,
-    and contain substantially more information
-    than inferred ARGs without these unary nodes.
-    We provide efficient algorithms to identify this ancestral haplotype information,
+    and contain substantially more information about inheritance % NSP-22NOV: "substantially more information" in what regard?
```

Comment:
this is vague, which is I think okay, but "substantially more represented ancestral haplotype" in the sense of "proprtion truth represented"